metadata-ingestion/src/datahub/configuration/validate_field_removal.py

```diff
@@ -0,0 +1,22 @@
+import warnings
+from typing import Type
+
+import pydantic
+
+
+def pydantic_removed_field(
```

Comment:
So basically this will ensure that Pydantic does not throw an exception in cases when a field has been removed? This makes a lot of sense!

Reply:
yup - this basically prints a warning and removes the field, so the remaining validators don't throw an error